local-dev/templates/firecloud-orchestration.conf.ctmpl

```diff
@@ -105,6 +105,14 @@ cwds {
     bucketName = "cwds-batchupsert-dev"
 }
 
+externalCreds {
+  baseUrl = "https://externalcreds.dsde-dev.broadinstitute.org"
+}
+
+foo {
+  bar = "baz"
+}
```

Comment:
did you mean to leave this here?

Reply:
whoops! nope. Was just testing that Orch could spin up with unknown keys just in case the terra-helmfile templates got released before the orch changes, and forgot to take it out!